### PR TITLE
New Port: Balatro card game

### PR DESCRIPTION
**Pull request by nkahoang**
# New Port for Balatro card game

## Game Information
- **Title**: Balatro
- **URL**: [{ Link to project page or source control page }](https://github.com/nkahoang/PortMaster-Balatro/blob/main/ports/balatro/README.md)

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] JELOS
Proof of running: https://www.reddit.com/r/SBCGaming/comments/1bgtqf2/guideshowcase_i_ported_balatro_to_the_rgb30/

### Resolution Tests
Test all major resolutions:
- [x] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Comment by Cebion** [comment]
Waiting for Testing in https://discord.com/channels/1122861252088172575/1221238851259994213
Looks promising.

**Comment by mhoogcarspel** [comment]
I would include a copy of the OFL license for the Nunito font used:
https://github.com/googlefonts/nunito/blob/main/OFL.txt

as we're distributing it as a font file, not embedded:
https://openfontlicense.org/how-to-use-ofl-fonts/

> The only situation in which an OFL font can be distributed without the text of the OFL (either in a separate file or in font metadata), is when a font is embedded in a document or bundled within a program.

**Review by kloptops — CHANGES_REQUESTED** [review]
Just needs arch change otherwise looks good.

File: ports/balatro/port.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        "desc": "Balatro is a poker-inspired roguelike deck builder all about creating powerful synergies and winning big.",
+        "inst": "On macOS, grab this Balatro.love file (~/Library/Application Support/Steam/steamapps/common/Balatro/Balatro.app/Contents/Resources/Balatro.love) and put into the balatro folder. On Windows, simply copy Balatro.exe to the balatro folder",
+        "genres": [
+            "card",
+            "puzzle"
+        ],
+        "image": {},
+        "rtr": false,
+        "runtime": null,
+        "reqs": []
```
[the remark this answers is not in the document]
Comment (nkahoang) [reply ports/balatro/port.json]: Fixed

**Review by JeodC — APPROVED** [review]
Looks good post-changes

**Review by JeodC — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Comment by mtrpires** [comment]
**Request to include r36s support to ports file**

[I've been trying to see](https://www.reddit.com/r/R36S/comments/1dlhgb4/goal_balatro_60fps_r36s/) whether Balatro can run at 60fps on my r36s and, thus far, have not been able to confirm it. I've tried editing some of the settings here and there, but I'm not sure I'm doing it right. 

I'd like to request adding support for the r36s and I could do it myself, but I would need guidance.

**Pull request merged**